Client/Frontend/ContextualHint/ContextualHintEligibilityUtility.swift

```diff
@@ -113,6 +113,7 @@ struct ContextualHintEligibilityUtility: ContextualHintEligibilityUtilityProtoco
             if cfrCounter <= 2, !hasOptedIn, hasTimePassed {
                 // - Display CFR-1
                 profile.prefs.setInt(cfrCounter + 1, forKey: PrefsKeys.ContextualHints.shoppingOnboardingCFRsCounterKey.rawValue)
+                profile.prefs.setTimestamp(Date.now(), forKey: PrefsKeys.FakespotLastCFRTimestamp)
```

Comment:
Can the tests for `canPresentShoppingCFR` be adjusted to represent this new addition?